docs/source/array-api.rst

```diff
@@ -25,6 +25,8 @@ Top level user functions:
    argwhere
    around
    array
+   asanyarray
```

Comment:
I'm pretty sure you need to add these to `__init__.py` to expose them at the top level.

Reply:
The already are in the `__init__.py`. https://github.com/dask/dask/blob/master/dask/array/__init__.py#L6

Reply:
Oh. In that case, I don't even know exactly what "public API" means :)

Reply:
Normally take that to mean documented. Though maybe that already should have happened when these got added.